### PR TITLE
MMM Case Study from PyData Global

### PR DESCRIPTION
**Pull request by juanitorduz**
Closes https://github.com/pymc-labs/pymc-marketing/issues/1014

----
📚 Documentation preview 📚: https://pymc-marketing--1044.org.readthedocs.build/en/1044/

### CODE REVIEW

File: pymc_marketing/mmm/mmm.py [hunk 1]
```diff
@@ -2159,6 +2160,10 @@ def allocate_budget_to_maximize_response(
             Custom constraints for the optimization. If None, no custom constraints are applied.
         quantile : float, optional
             The quantile to use for recovering transformation parameters. Default is 0.5.
+        noise_level : int
```
Comment (wd60622) [inline pymc_marketing/mmm/mmm.py:2163]:
```suggestion
        noise_level : float
```

Comment (wd60622) [reply pymc_marketing/mmm/mmm.py:2163]: Maybe a check for between 0 and 1

Comment (juanitorduz) [reply pymc_marketing/mmm/mmm.py]: Good catch! I will!

**Comment by juanitorduz** [comment]
Thanks @wd60622 ! I think all of the labels and legends are fixed now.

**Comment by wd60622** [comment]
Thanks for making that adjustment. Labels look better. Still have the comments on the notebook!

**Comment by juanitorduz** [comment]
> Thanks for making that adjustment. Labels look better. Still have the comments on the notebook!

Ups! I missed them sorry! I will work on them!

**Comment by juanitorduz** [comment]
<p>short and sweet ;)</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1044/discussion/#comment-2417938746'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by juanitorduz** [comment]
<p>yes! I like the idea</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1044/discussion/#comment-2417938774'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by juanitorduz** [comment]
<p>great point! added!</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1044/discussion/#comment-2397098357'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by juanitorduz** [comment]
<p>This is to compare about the zero value. I will clarify.</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1044/discussion/#comment-2417938794'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by juanitorduz** [comment]
<p>I added a more explicit explanation</p>

---

 <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1044/discussion/#comment-2417938864'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by juanitorduz** [comment]
ok, @wd60622, I think I addressed all of your comments; thanks! Let me know what you think about :)

**Review by wd60622 — APPROVED** [review]
Just the comment about the 80% then looks good! :rocket:

**Comment by juanitorduz** [comment]
Thank you very much for all the feedback! 🚀

**Pull request merged**